contracts/payloads/IGP44/PayloadIGP44.sol

```diff
@@ -0,0 +1,750 @@
+            setVaultLimitsAndAuth(VAULT_sUSDe_AND_USDC_USDT); // TYPE_3 => 40
+        }
+
+        {
+            // [TYPE 4] cbBTC-WBTC | cbBTC-WBTC | Smart collateral & smart debt
+            Vault memory VAULT_cbBTC_WBTC_AND_cbBTC_WBTC = Vault({
+                vault: getVaultAddress(41),
+                vaultType: TYPE.TYPE_4,
+                supplyToken: address(0),
+                borrowToken: address(0),
+                baseWithdrawalLimitInUSD: 40_000, // to be set later
+                baseBorrowLimitInUSD: 20_000, // to be set later
+                maxBorrowLimitInUSD: 25_000 // to be set later
+            });
+            setVaultLimitsAndAuth(VAULT_cbBTC_WBTC_AND_cbBTC_WBTC); // TYPE_4 => 41
+        }
+
+        {
+            // [TYPE 2] cbBTC-WBTC | USDC | Smart Collateral only
+            Vault memory VAULT_cbBTC_WBTC_AND_USDC = Vault({
+                vault: getVaultAddress(42),
+                vaultType: TYPE.TYPE_2,
+                supplyToken: address(0),
+                borrowToken: USDC_ADDRESS,
+                baseWithdrawalLimitInUSD: 40_000, // to be set later
+                baseBorrowLimitInUSD: 20_000, // to be set later
+                maxBorrowLimitInUSD: 25_000 // to be set later
+            });
+            setVaultLimitsAndAuth(VAULT_cbBTC_WBTC_AND_USDC); // TYPE_2 => 42
+        }
+
+        {
+            // [TYPE 2] cbBTC-WBTC | USDT | Smart Collateral only
+            Vault memory VAULT_cbBTC_WBTC_AND_USDT = Vault({
+                vault: getVaultAddress(43),
+                vaultType: TYPE.TYPE_2,
+                supplyToken: address(0),
+                borrowToken: USDT_ADDRESS,
+                baseWithdrawalLimitInUSD: 40_000, // to be set later
+                baseBorrowLimitInUSD: 20_000, // to be set later
+                maxBorrowLimitInUSD: 25_000 // to be set later
+            });
+            setVaultLimitsAndAuth(VAULT_cbBTC_WBTC_AND_USDT); // TYPE_2 => 43
+        }
+    }
+
+    /// @notice Action 2: Adding new lite implementation
+    function action2() internal {
+        // @TODO
+    }
+
+    /// @notice Action 3: Updating Liquidity User Module
+    function action3() internal {
+        {
+            bytes4[] memory sigs_ =
+                IProxy(address(LIQUIDITY)).getImplementationSigs(0x8eC5e29eA39b2f64B21e32cB9Ff11D5059982F8C);
+            IProxy(address(LIQUIDITY)).removeImplementation(0x8eC5e29eA39b2f64B21e32cB9Ff11D5059982F8C);
+
+            IProxy(address(LIQUIDITY)).addImplementation(
+                0x6967e68F7f9b3921181f27E66Aa9c3ac7e13dBc0,
+                sigs_
+            );
+        }
+    }
+
+    /// @notice Action 4: Adjusting rates for wstETH, WBTC & cbBTC
+    function action4() internal {
+        AdminModuleStructs.RateDataV2Params[] memory params_ = new AdminModuleStructs.RateDataV2Params[](3);
+
+        params_[0] = AdminModuleStructs.RateDataV2Params({
+            token: 0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0, // wstETH
```

Comment:
@0xprinc You can use from the above constant directly, right? 
![Uploading image.png…]()